docs/topics/ref_new-features-7-1-0.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.1.0/master.adoc
+
+
+:_newdoc-version: 2.18.2
+:_template-generated: 2024-07-01
+
+:_mod-docs-content-type: REFERENCE
+
+[id="new-features-7-1-0_{context}"]
+= New features
+
+This section provides the new features and improvements of the {ProductFullName} 7.1.0.
+
+.Support for analyzing applications managed with Gradle has been added to {ProductShortName} 
+
+Previously, you could use the {ProductFullName} to analyze Java applications managed only with Maven. With this update, {ProductShortName} can also extract dependencies from Gradle projects. As a result, you can now analyze applications that use Gradle instead of Maven. 
+
+[IMPORTANT]
+====
+Support for analyzing applications managed with Gradle is Developer Preview feature only. Developer Preview feature is not supported by Red{nbsp}Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview feature for production or business-critical workloads. Developer Preview feature provides early access to upcoming product feature in advance of its possible inclusion in a Red{nbsp}Hat product offering. Customers can use this feature to test functionality and provide feedback during the development process. This feature might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red{nbsp}Hat might provide ways to submit feedback on Developer Preview feature without an associated SLA.
+
+For more information about the support scope of Red{nbsp}Hat Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
+====
+
+
+
+.Support for analyzing .NET applications has been added to MTA
+
+In {ProductFullName} 7.1.0, you can use {ProductShortName} to analyze Windows-only .NET framework applications to aid migration from version 4.5 or later to multi-platform .NET 8.0 running on OpenShift Container platform.
+
+[IMPORTANT]
+====
+Support for analyzing .NET applications is Developer Preview feature only. Developer Preview feature is not supported by Red{nbsp}Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview feature for production or business-critical workloads. Developer Preview feature provides early access to upcoming product feature in advance of its possible inclusion in a Red{nbsp}Hat product offering. Customers can use this feature to test functionality and provide feedback during the development process. This feature might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red{nbsp}Hat might provide ways to submit feedback on Developer Preview feature without an associated SLA.
```

Comment:
If you use singular then you should use articles. Use plural and you can avoid
```suggestion
Support for analyzing .NET applications is a Developer Preview feature only. Developer Preview features are not supported by Red Hat in any way and are not functionally complete or production-ready. Do not use Developer Preview features for production or business-critical workloads. Developer Preview features provide early access to upcoming product features in advance of their possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. These features might not have any documentation, are subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview features without an associated SLA.
```

Reply:
Ooops, I guess it's just the matter or copy-pasting contents, lol :-D Haven't read the template properly yet -_-